apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/models/Sgv2ColumnDefinition.java

```diff
@@ -3,12 +3,14 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import io.quarkus.runtime.annotations.RegisterForReflection;
 import java.util.Objects;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
 // Copied from SGv1 ColumnDefinition
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @Schema(name = "ColumnDefinition")
+@RegisterForReflection
```

Comment:
Did you consider moving DTOs to records and getting rid of all that boilerplate code?

Reply:
Yes, there is a separate issue (#2264) for that. Just need to figure out its priorization.